### PR TITLE
Translated @FarAbout and @FarConfig help pages to English in FarEng.hlf.m4

### PR DESCRIPTION
**Pull request by MKadaner**
Please let me know if I should increment build number and/or update some other files.

### CODE REVIEW

File: far/FarEng.hlf.m4 [hunk 1]
```diff
@@ -5244,41 +5244,39 @@ $ #Macros: Autocompletion in dialogs#
...
 @FarAbout
-$ #Информация о версиях#
-    Запускается из командной строки #far:about#
-
-    Позволяет посмотреть:
-    - версию и битность Far Manager
-    - версии используемых в проекте сторонних библиотек
-    - имена и версии подключённых плагинов
+$ #Version invformation#
+    Starts from the command line #far:about#
```
Comment (alabuzhev) [inline far/FarEng.hlf.m4]: The initial line `Запускается из командной строки #far:about#` is poorly worded.
It should be something like `Запускается по команде #far:about#` / `Starts using the command #far:about#`.
Could you please update both versions?

File: far/FarEng.hlf.m4 [hunk 2]
```diff
@@ -5244,41 +5244,39 @@ $ #Macros: Autocompletion in dialogs#
...
+    Displays:
+    - Far Manager version and bitness
+    - versions of the third-party libraries used in the project
+    - names and versions of the active plugins
 
 @FarConfig
-$ #Редактор конфигурации#
-    Запускается из командной строки #far:config#
+$ #Configuration editor#
+    Starts from the command line #far:config#
```
Comment (alabuzhev) [inline far/FarEng.hlf.m4]: Same as above.

File: far/FarEng.hlf.m4 [hunk 3]
```diff
@@ -5244,41 +5244,39 @@ $ #Macros: Autocompletion in dialogs#
...
+the type (boolean, 3-state, integer, string), and the value (for the integer type, hexadecimal and symbolic representations additionally displayed).
+If current value of an option is other than the default, the option is marked with the '*' symbol to the left of the name.
 
-    Кроме клавиш перемещения по списку доступны следующие сочетания:
+    Besides the list navigation keys, the following key combinations are supported:
 
-    #Enter# или #F4#  изменить значение параметра,
-                  boolean и 3-state на месте,
-                  для integer и string вызывается диалог
+    #Enter# или #F4#  Change option value
```
Comment (alabuzhev) [inline far/FarEng.hlf.m4]: #Enter# **or** #F4# probably?

File: far/changelog [hunk 4]
```diff
@@ -1,3 +1,7 @@
+MKadaner 28.01.2018 15:50:33 -0800 - build 5134
```
Comment (alabuzhev) [inline far/changelog]: Please drop the ` - build 5134` part (since it's not changed).

File: far/changelog_eng [hunk 5]
```diff
@@ -3,6 +3,10 @@ This file is a translation of the main russian changelog and is provided by volu
 It might not always be as up to date as the main changelog.
 =======================================================================================
 
+MKadaner 28.01.2018 15:50:33 -0800 - build 5134
```
Comment (alabuzhev) [inline far/changelog_eng]: Same as above.

File: far/changelog (hunk 4 above)
Comment (MKadaner) [reply far/changelog]: Sure

File: far/changelog_eng (hunk 5 above)
Comment (MKadaner) [reply far/changelog_eng]: Done

File: far/FarEng.hlf.m4 (hunk 1 above)
Comment (MKadaner) [reply far/FarEng.hlf.m4]: Done. Thank you.

File: far/FarEng.hlf.m4 (hunk 2 above)
Comment (MKadaner) [reply far/FarEng.hlf.m4]: Done.

**Comment by alabuzhev** [comment]
Merged, thank you!

**Pull request merged**